packages/vscode/__tests__/ts-plugin-tests/smoketest-template-imports-ts-plugin.test.ts

```diff
@@ -55,6 +55,31 @@ describe('Smoke test: ETI Environment (TS Plugin Mode)', () => {
       ]);
     });
 
+    test.only('gives diagnostics for TypeScript file', async () => {
```

Comment:
This `.only()` needs to be removed so that other tests run. Normally vitest catches this on CI but these VSCode tests I think use mocha or something that doesn't catch cases like these.

Reply:
Bah. I missed this :(

Reply:
Ah sorry!